packages/components/CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 
 ### Enhancements
 
+-   `UnforwardedFontSizePicker`: Add `vw` and `vh` units to the font size picker ([#60507]((https://github.com/WordPress/gutenberg/pull/60607)).
```

Comment:
```suggestion
-   `FontSizePicker`: Add `vw` and `vh` units to the font size picker ([#60507]((https://github.com/WordPress/gutenberg/pull/60607)).
```

This PR "adds" a new unit and is available to users, so I think this section is fine.